sdk/include/fail-simulator-on-error.h

```diff
@@ -67,7 +67,7 @@ compartment_error_handler(ErrorState *frame, size_t mcause, size_t mtval)
 		DebugErrorHandler::log("Unhandled error {} at {}", mcause, frame->pcc);
 	}
 
-	simulation_exit(1);
+	(void)simulation_exit(1);
```

Comment:
As above, so far every call to this function has discarded the return result.  It either exits (and doesn't return) or doesn't exit (and we're not in a simulator that can exit).  We get no information from the non-void return.